web-client/src/modules/requests/constants.tsx

```diff
@@ -7,11 +7,16 @@ import * as Yup from 'yup';
 export const PostListLocation = new Location('/requests/list/:type', {
   type: Yup.string(),
 });
+// ?? todo remove Accepted and Open
+export const offersPostType = 'offers';
+export const offersLocationStr = '/requests/'+offersPostType;
```

Comment:
1. I thought the plan was to generalize and call it a `Post`. I understand your reasoning for not showing the word post to the end-users but I am not sure if this is the solution for it as in my opinion showing `requests` and then `offers` along with it would be even more confusing than showing `posts`.
2. If showing posts isn't an option, how about changing it to `list`?
3. Why have you chosen to go with this approach of constructing a string rather than using the location package? I don't think this is a good idea and is downgrading the quality of the code unless you have reasoning for it in which case could you explain that here?
4. Why have you chosen to remove and enum and instead use a variable? is there any advantage to doing it in this manner?

Reply:
A bit confused here, what's a Post? I'd agree with keeping just requests and offers, at least for the end-user (in the interface) cuz introducing new data structures might just be more confusing. The end-goal is: both a request and offer can be standalone waiting for someone to accept it AND both can be responses to existing requests and offers.